plugins/connectrpc/go/v1.15.0/buf.plugin.yaml

```
@@ -0,0 +1,20 @@
+version: v1
+name: buf.build/connectrpc/go
+plugin_version: v1.15.0
+source_url: https://github.com/connectrpc/connect-go
+integration_guide_url: https://connectrpc.com/docs/go/getting-started
+description: Generates client and server stubs for connectrpc.com/connect. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
+deps:
+  - plugin: buf.build/protocolbuffers/go:v1.32.0
+output_languages:
+  - go
+registry:
+  go:
+    min_version: "1.19"
```

```suggestion
    min_version: "1.20"
```

Ref: https://github.com/connectrpc/connect-go/blob/51a33b8cdae1bf32587a0d51a586a787cff7fa37/go.mod#L3